.env.example

```
@@ -0,0 +1,17 @@
+# Azure App Configuration connection string
+APPCONFIG_CONNSTR=Endpoint=https://sportdeetsappconfig2.azconfig.io;Id=...;Secret=...
+
+# PostgreSQL connection string (host.docker.internal resolves to your host machine)
+PG_CONNSTR=Host=host.docker.internal;Port=5432;Username=postgres;Password=YOUR_PASSWORD_HERE
+
+# Azure service principal used by DefaultAzureCredential to resolve Key Vault
+# references that come through Azure AppConfig. Mirrors the prod k3s `azure-identity`
+# secret (see sports-data-config/app/base/apps/api/api-deployment.yaml).
+# Create a dedicated dev SP with "Key Vault Secrets User" on your vault.
+AZURE_TENANT_ID=
+AZURE_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AZURE_CLIENT_ID key should go before the AZURE_TENANT_ID key

(UnorderedKey)